packages/shared/lib/core/profile/actions/active-profile/loadAccounts.ts

```diff
@@ -14,13 +14,13 @@ export async function loadAccounts(): Promise<void> {
         if (accountsResponse) {
             const loadedAccounts: IAccountState[] = []
             for (const account of accountsResponse) {
-                await account.sync()
+                // await account.sync()
```

Comment:
I will leave it uncommented as it currently only way we sync. But will later be replaced by background syncing. We commented it out because the bug in wallet.rs is duplicating accounts whenever this is called.